gradlew

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
```

Comment:
I'm surprised to see all this stuff checked in to the build to run Gradle. Can't we install it like Ant or Maven or other build tools, and only have metadata related to our project build in the repository?